src/v2/view-builder/utils/LinksUtil.js

```diff
@@ -117,14 +117,15 @@ const getSignOutLink = (settings, options = {}) => {
 };
 
 // Use it to create a widget configured link in the absence of `cancel` object in remediation
-const getBackToSignInLink = (settings) => {
+const getBackToSignInLink = () => {
   return [
     {
+      'actionPath': 'cancel',
       'type': 'link',
       'label': loc('goback', 'login'),
       'name': 'go-back',
       // TODO: OKTA-381328 back to baseUrl only works for default login page
-      'href': settings?.get('baseUrl'),
+      //'href': settings?.get('baseUrl'),
```

Comment:
why this ? this can't be commented.. It will break `Back to sign in` link for various flows.

Reply:
I've removed `href` and added `actionPath` because they can't work together according to this code:
https://github.com/okta/okta-signin-widget/blob/56930aa682bf8e0aee294b5a1c8588c5b0d4c6a8/src/v2/view-builder/components/Link.js#L41-L57

Reply:
@denysoblohin-okta there is already a version for that `getSignOutLink`

Reply:
Another possible solution I see is to catch specific `idx.session.expired` error and only in this case trigger `cancel` action on click on 'Back to sign-in' , in other cases use href

Reply:
Reverted these changes. Will move fix for expired session to another PR